### DISCUSSION TITLE
Bug Report:

### DISCUSSION THREAD

**Discussion by d4navi0** [KadeArchive/Kade-Engine#432] [Q&A]
## Steps to reproduce:
Open a replay
Do NOT press any key

## I was trying to replicate the senpai glitch effect from roses remix, then I went to replays and tried to open a replay. Then, I didn't press any key, then I died.

https://user-images.githubusercontent.com/63800482/117869056-9afe5900-b29a-11eb-9ecb-c1b85ec6ce04.mp4

**Comment by isakube** [comment]
This isn't in an unmodified Kade Engine, so this could always just be the mod. I might know how to read .kadeReplays so if you could send that one in particular i could try to see if it has any data. Since the arrows aren't being hit, either the key presses didn't save correctly, or aren't being replicated correctly. I have issues with sustain notes being dropped, so perhaps this is a similar case

**Comment by Kade-github** [comment]
First, really, replays barely work lol. They're going to be overhauled anyways. 



> This isn't in an unmodified Kade Engine, so this could always just be the mod. I might know how to read .kadeReplays so if you could send that one in particular i could try to see if it has any data. Since the arrows aren't being hit, either the key presses didn't save correctly, or aren't being replicated correctly. I have issues with sustain notes being dropped, so perhaps this is a similar case

they're just .json files but renamed
